blocks/form/file.js

```diff
@@ -0,0 +1,56 @@
+function attachFiles(wrapper, fileList, fileTemplate) {
+    for(let i = 0; i < fileList.length; i++) {
+        const file = fileList[i];
+        const input = fileTemplate.cloneNode(true);
+        input.id = input.name = 'attachment_' + Date.now() + '' + Math.floor((Math.random() * 1000));
+        if (input.accept.includes(file.type)) {
+            const dataTransfer = new DataTransfer();
+            dataTransfer.items.add(file);
+            input.files = dataTransfer.files;
```

Comment:
I am surprised that this works.

Reply:
Need to validate this usecase only with submit service.